docs/content/direct/multi-wec-aggregated-status.md

```diff
@@ -0,0 +1,160 @@
+
+## Overview
+
+KubeStellar propagates workload objects from a WDS to one or more WECs according to the bindings specified by the user. When a workload is downsynced to exactly one WEC, users may request singleton reported state, which copies the `.status` field from that WEC to the corresponding object in the WDS.
+
+When a workload is downsynced to multiple WECs, the default behavior leaves the `.status` field of the WDS object empty. Users can inspect the `CombinedStatus` objects for full per-cluster details, but the workload object itself does not show a summarized readiness or health view.
+
+The Multi-WEC Aggregated Status option allows users to request status aggregation in these multi-cluster cases. The aggregated status is written into the `.status` field of the workload object in the WDS using deterministic rules.
+
+## Enabling Multi-WEC Status Reporting
+
+Status return is configured in the `DownsyncModulation` section of a `BindingPolicy` or `Binding`. To request aggregated status from multiple WECs, set:
+
+```yaml
+wantMultiWECReportedState: true
+```
+
+This option parallels `wantSingletonReportedState` but applies in the case where more than one WEC is selected.
+
+Example:
+
+```yaml
+apiVersion: policies.kubestellar.io/v1alpha1
+kind: BindingPolicy
+metadata:
+  name: multiwec-nginx
+spec:
+  clusterSelectors:
+    - matchLabels:
+        location: edge
+  downsync:
+    - objectSelectors:
+        - matchLabels:
+            app.kubernetes.io/name: nginx-multi
+      wantMultiWECReportedState: true
+```
+
+For field definitions, see the DownsyncModulation API:
+https://github.com/kubestellar/kubestellar/blob/v0.29.0/api/control/v1alpha1/types.go#L138-L167
+
+## Behavior Summary
+
+The effect of status return options depends on the number of selected WECs.
+
+| `wantSingletonReportedState` | `wantMultiWECReportedState` | WEC count | Result |
+|:--:|:--:|:--:|:--|
+| T | F | 1 | Copy status from the single WEC. |
+| T | F | 0 or >1 | Clear `.status`. |
+| F | T | 1 | Copy status from the single WEC. |
+| F | T | >1 | Aggregate status from all WECs. |
+| T | T | 1 | Copy status from the single WEC. |
+| T | T | >1 | Aggregate status from all WECs (multi-WEC takes precedence). |
+| F | F | any | Leave `.status` empty. |
+
+**Legend:** T = true, F = false
+
+When both flags are enabled, the multi-WEC aggregation behavior takes precedence if more than one WEC is selected.
+
+## Aggregated Status Semantics
+
+The `.status` field of a Kubernetes workload object is defined for a single cluster. When aggregating from multiple WECs, KubeStellar applies approximation rules. The aggregation logic distinguishes two broad categories:
+
+- Workload kinds for which Argo CD defines built-in health assessment.
+- All other kinds.
+
+For Argo CD’s health rules, see:
+https://argo-cd.readthedocs.io/en/stable/operator-manual/health/
```

Comment:
But that's just a buggy document, right? The real truth in the code is different, right? If so then cite both.

(e.g., the cited user-facing document does not mention Conditions, so does not explain why the initial implementation does anything with Conditions. Also, the cited document is clearly buggy because it speaks of "updated replicas" for a Kind that does not have such a status field.)